app/controllers/concerns/cache_concern.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+module CacheConcern
+  extend ActiveSupport::Concern
+
+  def render_with_cache(**options)
+    raise ArgumentError, 'only JSON render calls are supported' unless options[:json].present? || block_given?
```

Comment:
`options[:json].present?` returns false for an existing but empty Hash, thus erroring out for the instance info controller as all the logic is in the serializer